packages/perseus-core/src/utils/util.ts

```diff
@@ -21,6 +21,54 @@ const seededRNG: (seed: number) => RNG = function (seed: number): RNG {
     };
 };
 
+// Note: Previously called cyrb128
```

Comment:
It might be nice to link to the source on Stack Overflow in this comment.

Reply:
Added the reference :)